packages/web-components/@sfx/product/src/variant.ts

```diff
@@ -42,15 +42,13 @@ export default class Variant extends LitElement {
 
   render(): TemplateResult {
     const { variant } = this;
+    const img = this.type === 'image' ? html`<img
+    class="sfx-variant-image"
+    src="${variant.image}"
+    alt="${variant.text}"
+  />` : '';
```

Comment:
Yes, this is what I meant, but I think we can improve the formatting. Maybe something like this?
```js
    const img = this.type === 'image'
      ? html`<img
          class="sfx-variant-image"
          src="${variant.image}"
          alt="${variant.text}"
        />`
      : '';
```